cmd/mailslurper/config.json

```diff
@@ -1,16 +1,18 @@
 {
 	"wwwAddress": "localhost",
-	"wwwPort": 8080,
+	"wwwPort": 8081,
 	"serviceAddress": "localhost",
 	"servicePort": 8085,
+	"serviceURL": "",
 	"smtpAddress": "localhost",
 	"smtpPort": 2500,
 	"dbEngine": "SQLite",
 	"dbHost": "",
 	"dbPort": 0,
-	"dbDatabase": "./mailslurper.db",
+	"dbDatabase": "/tmp/mailslurper.db",
```

Comment:
I'm not sure about changing the default location. And will this work on Windows?